data/themes/default-theme.yml

```diff
@@ -173,8 +173,11 @@ description_list:
 outline_list:
   indent: $horizontal_rhythm * 1.25
 table:
+  head_font_color: 000000
+  head_background_color: FFFFFF
   background_color: transparent
-  background_color_alt: f9f9f9
+  background_color_row_even: FFFFFF
+  background_color_row_odd: FFFFFF
```

Comment:
Let's use these names instead:

```
even_row_background_color: FFFFFF
odd_row_background_color: FFFFFF
```

That makes it consistent with `head_background_color` and so forth.